betty/model/association.py

```diff
@@ -311,72 +423,216 @@ def new_attr(self, instance: _OwnerT & Entity) -> EntityCollection[_AssociateT]:
         )
 
 
+class _RequiredToOneAssociation(
+    Generic[_OwnerT, _AssociateT],
+    _ToOneAssociation[
+        _OwnerT,
+        _AssociateT,
+        Intersection[_AssociateT, Entity],
+        RequiredResolvableEntity[Intersection[_AssociateT, Entity]],
+    ],
+):
+    _required = True
+
+    @override
+    def new_attr(self, instance: _OwnerT & Entity) -> Never:
```

Comment:
new_attr() is requires by Attr but not needed for requires to-one relationships. Should we remote the method, and by extension the entire Attr API?